py/desisim/scripts/quickquasars.py

```diff
@@ -55,7 +56,13 @@ def parse(options=None):
     parser.add_argument('--wmax', type=float, default=10000,help="Max wavelength (obs. frame)")
     parser.add_argument('--dwave', type=float, default=0.2,help="Internal wavelength step (don't change this)")
     parser.add_argument('--nproc', type=int, default=1,help="number of processors to run faster")
-    parser.add_argument('--zbest', action = "store_true" ,help="add a zbest file per spectrum with the truth")
+    
+    parser.add_argument('--zbest', action = "store_true",help="add a zbest file per spectrum either with the truth redshift or adding some error (use --sigma_kms_rsd and/or --sigma_kms_zfit)")
+
+    parser.add_argument('--sigma_kms_rsd',nargs='?',type=float,const=150, help="Adds a gaussian error to the quasar redshift thats simulate the fingers of god effect (default=150 km/s)")
+
+    parser.add_argument('--sigma_kms_zfit',type=float,help="Adds a gaussian error to the quasar redshift. This error could reflect things like peculiar velocities of the BH,outflows/orientation of the accretion disk/beam of quasar, and noise in spectrum that makes the identification of lines difficult. A suggested values is around 200 km/s, default is 0. ")
```

Comment:
Actually, thinking more about this, I think it is fine to have a default value of 400 for sigma_kms_zfit, as long as we ONLY apply it when we generating a zbest file, i.e., when we want to emulate a run of redrock. 

If we are not using --zbest option, then I would NOT add random errors to simulate redshift error fitting, since we would already "add" this noise when doing the actual redshift fit with redrock.